Dockerfile

```diff
@@ -16,9 +16,9 @@ ENV NODE_ENV production
 
 ADD package.json package-lock.json /app/
 RUN npm install --omit=dev
-ADD build/src /app/
+ADD src /app/
 
 # Cache of copy of the STAC JSON schemas by triggering a validation run
-RUN node /app/index.js stac-validate https://nz-imagery.s3-ap-southeast-2.amazonaws.com/new-zealand/new-zealand_2020-2021_10m/rgb/2193/collection.json
+RUN node /app/index.ts stac-validate https://nz-imagery.s3-ap-southeast-2.amazonaws.com/new-zealand/new-zealand_2020-2021_10m/rgb/2193/collection.json
 
-ENTRYPOINT ["node", "/app/index.js"]
```

Comment:
This changes will require to update all the topo-workflow entry point

Reply:
I wonder if we could avoid specifying the entrypoints in topo-workflow and use the defaults :shrug:

Reply:
looks like entrypoints are not required, so I have removed them in this pull request https://github.com/linz/topo-workflows/pull/985